app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt

```diff
@@ -86,38 +86,33 @@ fun HomeScreenContent(title: String) {
         )
         Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
         SearchBar()
-        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
 
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .verticalScroll(rememberScrollState())
-        ) {
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Sistemas", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-            TextBoxCarreras(text = "LICENCIATURA EN BIOLOGÍA", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Informática", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Industrial", onClick = {})
-            TextBoxCarreras(text = "Licenciatura en Ing en Electrónica", onClick = {})
-        }
 
     }
 }
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun SearchBar() {
+    val carreras = listOf(
+        "Licenciatura en Ing en Sistemas",
+        "Licenciatura en Ing en Informática",
+        "Licenciatura en Ing en Industrial",
+        "Licenciatura en Ing en Electrónica",
+        "LICENCIATURA EN BIOLOGÍA",
+        "Matematica",
+        "Fisica",
+        "Quimica",
+        "Programacion",
+        "Ingles",
+        "ELementos de la programacion",
+        "Estadistica l",
+
+        )
     var searchText by remember { mutableStateOf("") }
+    val filteredCarreras = carreras.filter { carrera ->
```

Comment:
never do a calculation inside a composable , in this case we dont have viewmodels , we would visti this part with viewmodels, nothing to correct just a warning for the futre